README.md

```diff
@@ -1,42 +1,37 @@
-<h1 align="center">
-  <img align="top" width="44" src="https://raw.githubusercontent.com/wxt-dev/wxt/HEAD/docs/public/hero-logo.svg" alt="WXT Logo">
-  <span>WXT</span>
-</h1>
-
-<p align="center">
-  <a href="https://www.npmjs.com/package/wxt" target="_blank"><img alt="npm version" src="https://img.shields.io/npm/v/wxt?labelColor=black&color=%234fa048"></a>
-  <span> </span>
-  <a href="https://www.npmjs.com/package/wxt" target="_blank"><img alt="downloads" src="https://img.shields.io/npm/dm/wxt?labelColor=black&color=%234fa048"></a>
-  <span> </span>
-  <a href="https://github.com/wxt-dev/wxt/blob/main/LICENSE" target="_blank"><img alt="license | MIT" src="https://img.shields.io/npm/l/wxt?labelColor=black&color=%234fa048"></a>
-  <span> </span>
-  <a href="https://codecov.io/github/wxt-dev/wxt" target="_blank"><img alt="coverage" src="https://img.shields.io/codecov/c/github/wxt-dev/wxt?labelColor=black&color=%234fa048"></a>
-</p>
-
-<p align="center">
-  <span>Next-gen framework for developing web extensions.</span>
-  <br/>
-  <span>⚡</span>
-  <br/>
-  <q><i>It's like Nuxt, but for Web Extensions</i></q>
-</p>
-
-<p align="center">
-  <a href="https://wxt.dev/guide/installation.html" target="_blank">Get Started</a>
-  &bull;
-  <a href="https://wxt.dev/api/config.html" target="_blank">Configuration</a>
-  &bull;
-  <a href="https://wxt.dev/examples.html" target="_blank">Examples</a>
-  &bull;
-  <a href="https://github.com/wxt-dev/wxt/blob/main/packages/wxt/CHANGELOG.md" target="_blank">Changelog</a>
-  &bull;
-  <a href="https://discord.gg/ZFsZqGery9" target="_blank">Discord</a>
-</p>
```

Comment:
I use HTML here to center everything horizontally:

<img width="925" alt="Screenshot 2025-02-12 at 9 01 27 AM" src="https://github.com/user-attachments/assets/bcbdbf43-3528-4d4e-9f04-0e8eaf97f8ed" />

These changes break that formatting:

<img width="1024" alt="Screenshot 2025-02-12 at 9 00 05 AM" src="https://github.com/user-attachments/assets/78515eeb-6538-419b-95c9-ffaa1fee2603" />

I would like to keep them centere, I think it looks better. So can we revert this section? Just the heading down to the bullet-separated links.

Reply:
Nope, everything is correct here

![image](https://github.com/user-attachments/assets/50b33998-0d88-49a2-bfe4-d9dfaeab0700)

Reply:
I see, guess it was just the PR preview that messed it up!